assets/js/atomic/blocks/product-elements/title/index.ts

```diff
@@ -36,15 +37,16 @@ const blockConfig: BlockConfiguration = {
 				__experimentalFontFamily: true,
 			},
 		} ),
-		...( isFeaturePluginBuild() && {
-			color: {
-				text: true,
-				background: true,
-				link: false,
-				gradients: true,
-				__experimentalSkipSerialization: true,
-			},
-		} ),
+		...( isFeaturePluginBuild() &&
+			hasSpacingStyleSupport() && {
```

Comment:
@gigitux Isn't that check should be used for spacing below?

Reply:
Update! Thanks for the double-check!